src/NemoStuff.jl

```diff
@@ -54,8 +54,6 @@ function identity_matrix(::Type{MatElem}, R::Ring, n::Int)
     return identity_matrix(R, n)
 end
 
-dense_matrix_type(::Type{T}) where {T} = Generic.MatSpaceElem{T}
```

Comment:
Whats up with this one?

Reply:
Duplicate

Reply:
The same already is in generic/Matrix.jl:38